views/js/pciCreator/dev/mathEntryInteraction/creator/widget/Widget.js

```diff
@@ -0,0 +1,17 @@
+define([
```

Comment:
File header missing : what is the license of this PCI ?
This comment is applicable to other js files too.